build.sh

```diff
@@ -2,4 +2,3 @@
 set -euo pipefail
 . debian/build.sh "$1" "$2"
 . docker/build.sh "$1" "$2" "full"
-
```

Comment:
Allgemein ist empfohlen, Dateien mit einem newline zu beenden.
https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
hab an den newlines nichts geändert. Nutze aber vscode auch zum arbeiten

Reply:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files
Würde ich rückgängig machen bzw. konsequent welche hinzufügen.